p2p/server.go

```diff
@@ -448,18 +445,7 @@ func (srv *Server) Stop() {
 	}
 	close(srv.quit)
 	srv.lock.Unlock()
-
-	stopChan := make(chan struct{})
```

Comment:
The previous code tries to skip `srv.loopWG.Wait()` if `stopTimeout` reach, maybe this is to solve issues before?
Is it ok just remove above `case <-cs.handler.stopCh:`, and keep this logic unchanged?